docs/application/web/guides/connectivity/bluetooth.md

```diff
@@ -734,12 +735,57 @@ To read and write a value of the Bluetooth descriptor:
       - write about start()
  -->
 ### Start the server
+
+To start the local GATT server:
+
+1. Retrieve `BluetoothGATTServer` object (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothGATTServer), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothGATTServer) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothGATTServer) applications) with `getGATTServer()` method of the `BluetoothManager` interface (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothManager), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothManager) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothManager) applications):
+
+```
+var server = tizen.bluetooth.getGATTServer();
+```
+
+2. Set up callbacks and start the local gatt server with `start()` method of `BluetoothGATTServer` (in [mobile](../../api/latest/device_api/mobile/tizen/bluetooth.html#BluetoothGATTServer), [wearable](../../api/latest/device_api/wearable/tizen/bluetooth.html#BluetoothGATTServer) and [tv](../../api/latest/device_api/tv/tizen/bluetooth.html#BluetoothGATTServer) applications) interface:
+
+```
+var onSuccess = function() {
+   console.log("Server started successfully");
+};
+
+var onError = function(e) {
+   console.error("Failed to start the server, error: " + e);
+};
+
+server.start(onSuccess, onError);
+```
+
 <!-- END #08 -->
```

Comment:
This TODO comment can be removed.

Reply:
All TODOs will be removed in the last commit.